test_data/asm/block_to_block.asm

```diff
@@ -22,7 +22,7 @@ machine Main with
 {
     Arith arith(N, N);
 
-    // return `3*x + 3*y`, adding twice locally and twice externally
```

Comment:
I think this was correct, we start from 0 and add three times in each block?

Reply:
Right!